analyses/bgd/README.md

```diff
@@ -1,33 +1,81 @@
 # Bangladesh Anticipatory Action
 
-## Background information
+[![Generic badge](https://img.shields.io/badge/STATUS-ENDORSED-%231EBFB3)](https://shields.io/)
```

Comment:
Could you add a 'Last updated' line to go along with this?

Reply:
Oops yes, done! And I also mentioned the activation